extensions/wikia/CommunityHeader/CommunityHeader.setup.php

```diff
@@ -10,7 +10,7 @@
 $wgHooks['MessageCacheReplace'][] = 'CommunityHeaderHooks::onMessageCacheReplace';
 
 // Controllers
-$wgAutoloadClasses['CommunityHeaderController'] = $dir . 'CommunityHeaderController.class.php';
+$wgAutoloadClasses['CommunityHeaderService'] = $dir . 'CommunityHeaderService.class.php';
```

Comment:
just remember that if you use service - you need to call it explicitly when rendering in template! :)